Source/WebCore/dom/FragmentDirectiveGenerator.cpp

```diff
@@ -121,6 +140,9 @@ void FragmentDirectiveGenerator::generateFragmentDirective(const SimpleRange& te
     VisiblePosition visibleStartPosition = VisiblePosition(Position(textFragmentRange.protectedStartContainer(), textFragmentRange.startOffset(), Position::PositionIsOffsetInAnchor));
     VisiblePosition visibleEndPosition = VisiblePosition(Position(textFragmentRange.protectedEndContainer(), textFragmentRange.endOffset(), Position::PositionIsOffsetInAnchor));
 
+    VisiblePosition visiblePrefixStartPosition = beforeStartOfCurrentBlock(visibleStartPosition);
```

Comment:
Is `visiblePrefixStartPosition` accurate? (If I'm reading this correctly, it would be the _end_ of the prefix range…)